### ISSUE TITLE
Outgoing Messages: Error: SMTP connection failure: SMTP failed to connect: io: timed out: timed out

### ISSUE THREAD

**Issue by x011** [deltachat/deltachat-desktop#2969]
- Operating System (Linux/Mac/Windows/iOS/Android): `Windows 11`
- Delta Chat Version: `1.32.1 (git: v1.30.0-65-gd0fedbc0)`
- Expected behavior: `send messages`
- Actual behavior: `cannot send messages`

I've been using dc on windows 11 for several years already but suddenly I cannot send messages anymore.
I checked the postfix logs on my server and see this:

```
Oct 29 13:27:44 ns1 postfix/smtps/smtpd[19964]: connect from static.xxxxx.clients.your-server.de[x.x.x.x]
Oct 29 13:28:14 ns1 postfix/smtps/smtpd[19964]: SSL_accept error from static.xxxxx.clients.your-server.de[x.x.x.x]: lost connection
Oct 29 13:28:14 ns1 postfix/smtps/smtpd[19964]: lost connection after CONNECT from static.xxxxx.clients.your-server.de[x.x.x.x]
Oct 29 13:28:14 ns1 postfix/smtps/smtpd[19964]: disconnect from static.xxxxx.clients.your-server.de[x.x.x.x]
```

![image](https://user-images.githubusercontent.com/4313821/198829437-5711468e-fdf2-43ac-acd6-005a9d190d17.png)

SSL/TLS seems to be working normally on the server and I can send messages using dc on my mobile, also tested `openssl s_client -connect x.x.x.x:465` and `openssl s_client -starttls smtp -connect  x.x.x.x:25` without  errors.

Any idea how to fix this?

**Comment by x011** [comment]
I tried re-configuring the account as advised with the same settings (1) that I use on Thunderbird, and always used on dc, but the configuration wizard (2) don't show any progress or error message. I waited 10 minutes until cancelling the process.

(1)
![1](https://user-images.githubusercontent.com/4313821/198878178-aa7bfe6d-4e53-4633-9c9f-d54b7a41a651.png)

(2)
![image](https://user-images.githubusercontent.com/4313821/198878216-3bf2acee-a849-44ff-8415-5d755cb9466e.png)



Notes:

-  on the postfix logs I can see dc trying connecting multiple times and this leads to a [temporary ban](https://serverfault.com/a/422767/92233) (too many errors without sending a message)
- Thunderbird and dc are using the same computer and network.
-  Logs sent to [delta[a]merlinux.eu](mailto:delta[a]merlinux.eu)
